Formula/sc-im.rb

```diff
@@ -0,0 +1,20 @@
+class ScIm < Formula
+  desc "Spreadsheet program for the terminal, using ncurses"
+  homepage "https://github.com/andmarti1424/sc-im"
+  url "https://github.com/andmarti1424/sc-im/archive/v0.5.0.tar.gz"
+  sha256 "d7a31c5225e02239e925b50b414d7e69d12bc3554f218621823782872ccc5e4d"
+  head "https://github.com/andmarti1424/sc-im.git"
+
+  depends_on "ncurses"
+
+  def install
+    cd "src" do
+      system "make", "prefix=#{prefix}", "LDFLAGS=-L#{Formula["ncurses"].lib}"
+      system "make", "prefix=#{prefix}", "install"
+    end
```

Comment:
I actually looked into what Homebrew's preference was regarding using a `cd` block vs a `-C` flag. Based on some rough grepping, it seemed like `cd` was used about twice as often as `-C` when calling `make` more than once in that directory.

I'm fine changing it, but I thought the way it's written currently was more idiomatic Homebrew and more readable.